debugAdapterProtocol.json

```diff
@@ -392,6 +392,10 @@
 							"data": {
 								"type": [ "array", "boolean", "integer", "null", "number" , "object", "string" ],
 								"description": "Additional data to report. For the `telemetry` category the data is sent to telemetry, for the other categories the data is shown in JSON format."
+							},
+							"locationReference": {
+								"type": "string",
+								"description": "A reference that allows the client to request the location where the new value is declared. For example, if the logged value is function pointer, the adapter may be able to look up the function's location. This should be present only if the adapter is likely to be able to resolve the location.\n\nThis reference shares the same lifetime as the `variablesReference`. See 'Lifetime of Object References' in the Overview section for details."
```

Comment:
> This should be present only if

Does the value 0 have any special meaning? Or is it a valid `locationReference`? For comparison, for `variablesReference`, the value 0 counts as "non-present":

> If an attribute `variablesReference` exists and its value is > 0 [...]

Reply:
I don't think so. The output event is actually the only place I see where say `variablesReference` is treated specially. I would rather avoid adding such special casing to a new reference type